modules/base/include/modules/base/algorithm/image/layercontour.h

```diff
@@ -40,10 +40,8 @@ namespace inviwo {
 class LayerRepresentation;
 class Mesh;
 
-class IVW_MODULE_BASE_API ImageContour {
-public:
-    static std::shared_ptr<Mesh> apply(const LayerRepresentation* in, size_t channel,
-                                       double isoValue, vec4 color = vec4(1.0));
-};
+IVW_MODULE_BASE_API std::shared_ptr<Mesh> computeLayerContour(const LayerRepresentation* in,
```

Comment:
what about more then one contour? would be nice to get them the same mesh right?

Reply:
separate PR?

Reply:
sure